spec/controllers/spree/admin/reports_controller_spec.rb

```diff
@@ -68,10 +68,13 @@
 
     describe 'Orders & Fulfillment' do
       it "shows all orders in order cycles I coordinate" do
-        spree_get :orders_and_fulfillment
+        # create test objects
+        [orderA1, orderA2, orderB1, orderB2]
```

Comment:
why not using `let!` instead of this expression? seems a bit odd to me. The fact that you need a comment feels like a symptom to me.

Reply:
The thing about the comment is true. I don't know of any pattern for this. If I used `let!`, would need to add a whole block around every single example so that the `let!` happens only for that one example. Or can you use it within `it`?

I would like to create these objects on a case by case basis, individually for each `it`. I could assign them to a variable just for naming purposes, but then we have an unused variable: `test_context = [orderA1, orderA2, orderB1, orderB2]`

How would you do it?